src/components/organisms/Tabs/TabButtons/TabButtons.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, FC, ChangeEvent } from 'react';
+import { useTabsContext } from '../hooks/useTabsContext';
+import { useViewport } from '../../../../hooks/useViewport';
+import Dropdown from '../../../atoms/Dropdown/Dropdown';
+import { grid } from '../../../../constants';
+import TabButton, { TabButtonProps } from '../TabButton/TabButton';
+import { DropdownOption } from '../../../..';
+
+interface TabButtonsProps {
+  tabButtons: TabButtonProps[];
+  defaultTab?: string;
+}
+
+const TabButtons: FC<TabButtonsProps> = ({ tabButtons, defaultTab }) => {
+  const { width = 0 } = useViewport();
+
+  const { setActiveTab } = useTabsContext();
+
+  useEffect(() => {
+    defaultTab && setActiveTab(defaultTab);
+  }, []);
+
+  return (
+    <>
+      {tabButtons.length > 2 && width <= grid.breakpoints.s ? (
+        <Dropdown
+          defaultValue={defaultTab}
+          onChange={(event: ChangeEvent<HTMLSelectElement>) => setActiveTab(event.target.value)}
+        >
+          {tabButtons.map((tabButton: TabButtonProps) => (
```

Comment:
this `: TabButtonProps` shouldn't be necessary as it's already inferred by the compiler when you declared `tabButtons: TabButtonProps[];` 

Same in line 37